packages/core/src/App/Containers/RealAccountSignup/currency-selector.scss

```diff
@@ -129,6 +129,7 @@
             margin: 0.4rem 0 0;
             font-weight: bold;
             color: var(--text-prominent);
+            text-align: center;
```

Comment:
Can we remove `font-size`, `font-weight`, `color` and `text-align` and use the `Text` component instead?

Reply:
> Can we remove `font-size`, `font-weight`, `color` and `text-align` from here and use the `Text` component instead?
Sure will do

Reply:
> Can we remove `font-size`, `font-weight`, `color` and `text-align` from here and use the `Text` component instead?

Reply:
Done